main/reference/ertp-api/amount-math.md

```diff
@@ -296,17 +237,40 @@ const subtractedList = AmountMath.subtract(listAmountA, listAmountC)
 // Throws error
 const badList = AmountMath.subtract(listAmountA, listAmountB)
 ```
+
+## AmountMath.min(x, y, brand?)
```

Comment:
All the other functions have code samples. 

I would give samples with fungibles (quatloos are fine).  Make two amounts, and get the min here and the max below.  I think it's worth giving working examples for non-fungibles. You've done it with seats before, and that would be fine here. I would go ahead and write it up as if the bug below has been fixed. That means for pairs of non-fungibles that have a greater-than relationship, they return the appropriate one. For pairs that don't, they throw.

I've filed a bug against [min/max for non-fungibles](https://github.com/Agoric/agoric-sdk/issues/6691).

Reply:
I added code for fungibles, but I realized that I don't know how non-fungibles are actually compared? For example, for stadium seats, is "Row 2, Seat 35" greater or less than "Row 4, Seat 1"? Where might such a comparison be defined?

Reply:
The applicable AmountMath is set arithmetic. It's not about comparing elements. Set A is greater than Set B if it contains elements that aren't in Set B and Set B doesn't contain any elements that aren't in Set A.

If the designer of an NFT wants to treat some items that could appear in purses as "better" or more "valuable" than others, they'd have to do this with some other helper methods.

The AmountMath methods are only there to help determine whether you could subtract one from the other. It doesn't make any sense to subtract seat "Row 2, Seat 35" from "Row 4, Seat 1", so neither is greater than the other.